website/docs/d/google_folder.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "google"
+page_title: "Google: google_folder"
+sidebar_current: "docs-google-datasource-folder"
+description: |-
+  Get information about a Google Cloud Folder.
+---
+
+# google\_folder
+
+Use this data source to get information about a Google Cloud Folder.
+
+```hcl
+# Get folder by id
+data "google_folder" "my_folder_1" {
+  folder = "folders/12345"
+  lookup_organization = true
+}
+
+# Search by fields
+data "google_folder" "my_folder_2" {
+  folder = "folders/23456"
+}
+
+output "my_folder_1_organization" {
+  value = "${data.google_folder.my_folder_1.organization}"
+}
+
+output "my_folder_2_parent" {
+  value = "${data.google_folder.my_folder_2.parent}"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `folder` (Required) - The name of the Folder in the form `{folder_id}` or `folders/{folder_id}`.
+* `lookup_organization` (Optional) - `true` to find the organization that the folder belongs, `false` to avoid the lookup. It searches up the tree.
```

Comment:
Can you document whether it is set to true or false by default?

Reply:
done in new commit